deps.bzl

```diff
@@ -982,8 +982,8 @@ def prysm_deps():
         patches = [
             "//third_party:com_github_ethereum_go_ethereum_secp256k1.patch",
         ],
-        sum = "h1:uuBkYUJW9aY5JYi3+sqLHz+XWyo5fmn/ab9XcbtVDTU=",
-        version = "v1.11.3",
+        sum = "h1:3M1uan+LAUvdn+7wCEFrcMM4LJTeuxDrPTg/f31a5QQ=",
+        version = "v1.11.5",
```

Comment:
There seems to be a transitive dependency issue with `v1.11.6`. We use Marius' tx fuzzer, which uses holiman's fork of evmlab, which refers to a struct field that changed between 1.11.5->1.11.6.

Reply:
I think that has been updated. https://github.com/holiman/goevmlab/pull/91

Try
```
go get github.com/ethereum/go-ethereum github.com/holiman/goevmlab
bazel run //:gazelle -- update-repos -from_file=go.mod -to_macro=deps.bzl%prysm_deps -prune=true

# Builds OK
bazel build //testing/endtoend:presubmit //testing/endtoend:postsubmit
```